.gitignore

```diff
@@ -69,10 +69,7 @@ regulations/static/regulations/js/source/require.config.js
 regulations/static/regulations/js/source/lib/
 regulations/static/regulations/js/tests/browser/test.config.js
 regulations/static/regulations/js/built/*
-!regulations/static/regulations/js/built/regulations.js
-!regulations/static/regulations/js/built/require.config.js
-!regulations/static/regulations/js/built/lib/requirejs/require.js
-!regulations/static/regulations/js/built/lib/respond/respond.min.js
+regulations/static/regulations/js/built/regulations.js
```

Comment:
Since we have `built/*` on there, do we need to explicitly exclude regulations.js?